README.md

```diff
@@ -372,27 +389,41 @@ We use git-flow for naming and versioning conventions. Both the API and web app
     git flow hotfix start my-hotfix
     ```
 
-* Reviewer merges hotfix branch into develop and master and pushes to origin
-* [auto] Develop is deployed to dev
-* [auto] Master is deployed to prod
+* Reviewer merges hotfix branch into `develop` and `master` and pushes to `origin`:
+
+    ```
+    git flow hotfix finish my-hotfix
+    git checkout master
+    git push origin master --follow-tags
+    git checkout develop
+    git push origin develop
+    ```
+
+* `develop` is deployed to `dev`
+* `master` is deployed to `prod`
 
 ### Creating a release
-* Developer creates a release branch and pushes to origin:
+* Developer creates a release branch and pushes to `origin`:
 
     ```
     git flow release start my-release
-    git flow release publish my-release
+    git push origin release/my-release
```

Comment:
I ran into an issue with this last time I tried to use it, perhaps I had something misconfigured though.  I'll take another look.